packages/fxa-auth-server/lib/senders/emails/templates/subscriptionAccountReminderSecond/index.mjml

```diff
@@ -0,0 +1,27 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="subscriptionAccountReminderSecond-title">Welcome to Firefox!</span>
+    </mj-text>
+
+    <mj-text css-class="text-body">
+      <span data-l10n-id="subscriptionAccountReminderSecond-content-info">
+        A few days ago you created a Firefox account but never verified it. We hope you’ll finish setting up your account, so you can use your new subscription.
+      </span>
+    </mj-text>
+
+    <mj-text css-class="text-body">
+      <span data-l10n-id="subscriptionAccountReminderSecond-content-select">
+        Select “Create Password” to set up a new password and finish verifying your account.
+      </span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<%- include ('/partials/button/index.mjml', { cssClass: 'mb-8' }) %>
+<%- include ('/partials/subscriptionSupport/index.mjml') %>
+<%- include ('/partials/automatedEmailResetPassword/index.mjml') %>
```

Comment:
Piggybacking off my review on your other PR, this here + in some plaintext files _really_ makes me wonder if this is supposed to be in the HTML of every subplat email😅 Things to figure out later!

Reply:
I've added it to the Confirmation Needed doc. 🙂